CODE_OF_CONDUCT.md

```
@@ -1,78 +1,147 @@
+Perilaku lain yang secara wajar dapat dianggap tidak pantas dalam lingkungan profesional
+
+Tanggung Jawab Penegakan
+Pemimpin komunitas bertanggung jawab untuk mengklarifikasi dan menegakkan standar perilaku kami yang dapat diterima dan akan mengambil tindakan korektif yang tepat dan adil sebagai tanggapan terhadap perilaku apa pun yang mereka anggap tidak pantas, mengancam, menyinggung, atau berbahaya.
+
+Pemimpin komunitas memiliki hak dan tanggung jawab untuk menghapus, mengedit, atau menolak komentar, komitmen (commits), kode, editan wiki, masalah (issues), dan kontribusi lain yang tidak selaras dengan Kode Etik ini, dan akan mengkomunikasikan alasan keputusan moderasi bila sesuai.
+
+Ruang Lingkup
+Kode Etik ini berlaku di semua ruang komunitas, dan juga berlaku ketika seorang individu secara resmi mewakili komunitas di ruang publik. Contoh perwakilan komunitas kami meliputi penggunaan alamat email resmi, posting melalui akun media sosial resmi, atau bertindak sebagai perwakilan yang ditunjuk pada acara daring atau luring.
+
+Penegakan
+Insiden perilaku yang kasar, melecehkan, atau tidak dapat diterima lainnya dapat dilaporkan kepada pemimpin komunitas yang bertanggung jawab untuk penegakan di developer@kiiglobal.io. Semua keluhan akan ditinjau dan diselidiki dengan segera dan adil.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)